### DISCUSSION TITLE
本地挂载的视频无法播放

### DISCUSSION THREAD

**Discussion by aknyzsd** [AlistGo/alist#5325] [Q&A]
一直显示重新连接，加载不出来，但是可以下载

**Comment by anwen-anyi** [comment]
是不是编码不对？ 或者 在配置里面添加一下site_url

**Comment by xushulang** [comment]
> 一直显示重新连接，加载不出来，但是可以下载

我也有类似问题，视频有时候出现反复加载、有声音无画面、播放画面帧来回卡等等问题，不知为何
